app/Resources/views/base.html.twig

```diff
@@ -29,7 +29,7 @@
                             <span class="icon-bar"></span>
                             <span class="icon-bar"></span>
                         </button>
-                        <a class="navbar-brand" href="#">DashboardHub</a>
+                        <a class="navbar-brand" href="#">DashboardHub <span class="label label-warning version">Pre-Alpha</span></a>
```

Comment:
do you not want to put the contents of the <a> on a new line with indents for readability?